modules/benchmark/man/align.data.Rd

```diff
@@ -0,0 +1,19 @@
+% Generated by roxygen2 (4.0.2): do not edit by hand
```

Comment:
you need to update your version of roxygen and recompile the documentation. In the latest release the version number is no longer in every file.